src/_scss/pages/trainingVideos/_featuredVideo.scss

```diff
@@ -62,6 +54,10 @@
       @media(min-width: $tablet-screen){
         font-size: $font-size-40;
       }
+      &:hover {
+        cursor: pointer;
+          text-decoration: underline;
+        }
     }
 
```

Comment:
at line 81 in this file you can remove the width: 100%
the width: 24 is the one that's being applied